src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/TrancheOne.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.Booking
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.CalculationResult
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+@Component
+class TrancheOne(
+  @Value("\${sds-early-release-tranches.tranche-one-date}")
+  @DateTimeFormat(pattern = "yyyy-MM-dd") val trancheCommencementDate: LocalDate,
+) : Tranche {
+
+  override fun isBookingApplicableForTrancheCriteria(calculationResult: CalculationResult, booking: Booking): Boolean {
+    return booking.consecutiveSentences.none { consecutiveSentence ->
```

Comment:
So they're only tranche one if they have sentences less than 5 years. We don't calculate per sentence? What if the 5 year sentence isn't the sentence that provides the CRD and they have a concurrent 2 year sentence which provides the release date. Are they still tranche two?

Reply:
That's the logic 🙃